e2e/full_scan_test.go

```diff
@@ -76,6 +78,53 @@ var _ = ginkgo.Describe("Running a full scan (exploits, info finder, malware, mi
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
 				return len(*scans.Items) == 1
 			}, DefaultTimeout*10, time.Second).Should(gomega.BeTrue())
+
+			ginkgo.By("waiting until asset is found in riskiest assets dashboard")
+			gomega.Eventually(func() bool {
+				riskiestAssets, err := uiClient.GetDashboardRiskiestAssets(ctx)
```

Comment:
Are we sure that the scanned container will be a risky asset?
Also the question stands for finding a critical vulnerability in it on line 90.

Reply:
The alpine image scanned has a pinned version, so, the vulnerabilities found today will always be present. If we update the alpine version and no vulnerabilities are found the test will fail. However, we currently gate PRs with E2E tests